crates/ironrdp-session/src/image.rs

```diff
@@ -287,23 +287,23 @@ impl DecodedImage {
             _ => return,
         };
 
-        let left_virtual = x as i16 - pointer.hot_spot_x as i16;
-        let top_virtual = y as i16 - pointer.hot_spot_y as i16;
+        let left_virtual = x as i16 - pointer.hotspot_x as i16;
+        let top_virtual = y as i16 - pointer.hotspot_y as i16;
```

Comment:
Note: all the `as i16` here are conversions with possible "wrapping" (e.g.: `hotspot_y` > `i16::MAX`). I don’t remember what is around this code, but I suppose some operation _must_ be performed using unsigned integers? Would it make sense to use `i32::from` instead?

Reply:
Yeah, we need it here to check if the cursor sprite is partially outside the framebuffer, to cut its sides appropriately

Reply:
Sounds good! Fair enough.